### PR TITLE
Updated docs for pub/sub

### PR DESCRIPTION
**Pull request by jigyasak05**
Signed-off-by: Jigyasa <jigyasakhaneja05@gmail.com>

### CODE REVIEW

File: docs/artefacts/offramps.md [hunk 1]
```diff
@@ -872,6 +872,33 @@ offramp:
...
+  "command": "create_subscription",
+  "project": "<project-id>",
+  "topic": "<topic-name>",
+  "subscription": "<subscription-name>",
+  "message_ordering": `<message-ordering>`
+}
+```
+
+**Response**:
+```js
```
Comment (mfelsche) [inline docs/artefacts/offramps.md:891]: whoopsie, does pub/sub provide an empty response? or is the response payload missing?

**Review by mfelsche — COMMENTED** [review]
Otherwise LGTM

File: docs/artefacts/offramps.md (hunk 1 above)
Comment (jigyasak05) [reply docs/artefacts/offramps.md:891]: It does actually :'D I had left it as a TODO. As of now, it throws an error if the subscription doesn't get created and no response if it does. I'll fix this to give a response ^_^

Comment (jigyasak05) [reply docs/artefacts/offramps.md:892]: Modified the docs!

**Review by mfelsche — DISMISSED** [review]
Great! Lets wait with merging until the tremor PR ﻿https://github.com/tremor-rs/tremor-runtime/pull/1082 is merged :)

**Comment by Licenser** [comment]
The table at the beginning has `gcs ` but ismissing `gpub`

**Comment by jigyasak05** [comment]
> The table at the beginning has `gcs ` but ismissing `gpub`
Added 👍

**Review by Licenser — APPROVED** [review]
Wooohhh! :rocket: well done :D

**Pull request merged**